bin/crew

```diff
@@ -712,10 +712,8 @@ def download
         end
       end
       # Download file if not cached.
-      downloader url, filename, @opt_verbose
+      downloader url, sha256sum, filename, 3, @opt_verbose
```

Comment:
Should we make the retry count a variable?  It'd be nice if it was configurable.

Reply:
Now it can be adjust with `CREW_DOWNLOADER_RETRY` env variable

Reply:
Great work @supechicken.  Is this ready?

Reply:
> Great work @supechicken. Is this ready?

Yes

Reply:
Great, can we work on #6886 next? :)